### PR TITLE
Update ug dg

### PR DESCRIPTION
**Pull request by jelneo**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -246,14 +251,15 @@ Increases the debt of the 2nd person by $234.
...
 
 ****
 * Decreases the debt of the person at the specified `INDEX` by `AMOUNT`. The index refers to the index number shown in the last person listing. The index *must be a positive integer* 1, 2, 3, ...
-* If no index is specified, the debt of the currently selected person is decreased instead.
-* `AMOUNT` has to be in dollars and cents. For example: `600` which represents $600.
-* `AMOUNT` repaid cannot be more than the debt owed by the person at the specifiec `INDEX`
+* If no index is specified, the debt of the currently selected person is updated instead.
+* `AMOUNT` has to be in dollars and cents. For example: `600.15` which represents $600.15.
+* `AMOUNT` repaid cannot be more than the debt owed by the person at the specifiec `INDEX`.
+* If a person fully repays his/her debts, the date repaid will be set to the day that this command was entered. Also, the person will be listed in the whitelist.
```
Comment (khooroko) [inline docs/UserGuide.adoc]: not always. if a blacklisted person does this he/she will not be in the whitelist.
consider changing to "in the whitelist if he/she is not banned" or "not blacklisted"

Comment (jelneo) [reply docs/UserGuide.adoc]: corrected

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -366,52 +366,50 @@ _Figure 4.1.2g: `UndoRedoStack` activity diagram_
...
-The arguments (index and amount borrowed/paid back) are separated by a whitespace instead of special prefixes (e.g. prefix `p/` used for phone number). Hence, the `String#split` method is used to tokenize the input using a single whitespace as the delimiter. As seen in Figure 4.2.1, the tokenized inputs (index and amount borrowed) are then converted to their appropriate `Object` types and supplied as arguments to the `BorrowCommand` constructor. Input for the `payback` command is tokenized in the same manner.
+The arguments (index and amount borrowed/paid back) are separated by a whitespace instead of special prefixes (e.g. prefix `p/` used for phone number). Hence, the `String#split` method is used to tokenize the input using a single whitespace as the delimiter. As seen in Figure 4.2.1, the tokenized inputs (index and amount borrowed) are then converted to their appropriate `Object` types and supplied as arguments to the `BorrowCommand` constructor. Input for the `payback` command is tokenized and supplied to the `PaybackCommand` constructor in the same manner.
```
Comment (khooroko) [inline docs/DeveloperGuide.adoc]: we no longer have p/

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -366,52 +366,50 @@ _Figure 4.1.2g: `UndoRedoStack` activity diagram_
...
-`BorrowCommand` updates the `debt` and `totalDebt` attributes in the target `Person` object in the `AddressBook` class (which is in the `Model` component). A new `Person` object is created with the updated `debt` and `totalDebt` amount. The target `Person` object is then replaced with this new `Person` object.
-`PaybackCommand` updates the target person's debt in the same manner as the `BorrowCommand`. However, only the `debt` attribute is updated.
+`BorrowCommand` updates the `debt`, `totalDebt` and `dateRepaid` attributes in the target `Person` object through the `Model#addDebtToPerson()` which calls `AddressBook#addDebtToPerson()`. A new `Person` object is then created with the updated `debt` and `totalDebt` amount. Also, the `dateRepaid` field is set to `NOT REPAID`.  The target `Person` object is then replaced with this new `Person` object.
+
+`PaybackCommand` updates the target person's debt in the same manner as the `BorrowCommand`. However, only the `debt` attribute is updated. If the person has fully repaid his/her debts, the `PaybackCommand` will set the date repaid to the day the `PaybackCommand` was executed. The person will also be listed in the whitelist.
```
Comment (khooroko) [inline docs/DeveloperGuide.adoc]: i think you didn't see my prev comment? the person will not be listed in the whitelist if this is done on a blacklisted person

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -621,16 +619,16 @@ public class ClusterUtil {
 
 ==== Design Considerations
 
-**Aspect:** Grouping of contacts. +
-**Alternative 1 (current choice):** Create a field that contains the name and district number of the postal districts, based on postal code. +
+**Aspect:** Grouping of contacts +
+**Alternative 1 (current choice):** Create a field that contains the name and district number of the postal districts, based on postal code.\ +
```
Comment (khooroko) [inline docs/DeveloperGuide.adoc]: did you typo your backspace into "\\"?

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -684,21 +693,21 @@ When a user enters the `logout` command, the `LogoutCommand` class in the `Logic
...
+ * a character in the command box text field with a BLACK_CIRCLE
+ */
 if (numOfSpaces >= 2 && currentInput.charAt(currentInput.length() - 1) != ' '
         && currentInput.charAt(currentInput.length() - 1) != BLACK_CIRCLE) {
     maskPasswordInput(currentInput);
-
 }
 ----
-**Cons:** Although '\*' is an invalid password character, the user may still use it while entering the password. If the user types '*', the cursor will remain at its original position. The user will be under the impression that the command box is not registering what was typed.
+**Cons:** Although '\*' is an invalid password character, the user may still use it while entering the password. hence, if the user types '*', the cursor will remain at its original position. The user will be under the impression that the command box is not registering what was typed.
```
Comment (khooroko) [inline docs/DeveloperGuide.adoc]: Hence* with the capital H

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -834,12 +843,12 @@ public ReadOnlyPerson selectPersonForCommand(Index index) throws CommandExceptio
 
 ==== Design Considerations
 
-**Aspect:** Executing index-based commands without index. +
-**Alternative 1 (current choice):** These person to apply the commands on are determined in the constructors. +
+**Aspect:** Executing index-based commands without index +
+**Alternative 1 (current choice):** These person to apply the commands on are determined in the constructors +
```
Comment (khooroko) [inline docs/DeveloperGuide.adoc]: "The* person to" sorry my bad

**Review by khooroko — COMMENTED** [review]
UG section 4.9 (overduelist) show be "Shows" with the capital S instead of "show"

**Comment by khooroko** [comment]
Some UG examples do not have newlines as appropriate (my bad) 
Ban, unban, select (needs 2 newlines), nearby (`nearby` should not be a separate point from the above 2 commands), addpic, delpic,

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (jelneo) [reply docs/DeveloperGuide.adoc]: corrected

File: docs/DeveloperGuide.adoc (hunk 4 above)
Comment (jelneo) [reply docs/DeveloperGuide.adoc]: corrected

File: docs/DeveloperGuide.adoc (hunk 5 above)
Comment (jelneo) [reply docs/DeveloperGuide.adoc]: corrected

**Comment by khooroko** [comment]
DG: 
In user stories, two-FA should be changed to 2FA because it's not linking to the glossary (or change 2FA to two-FA in glossary)
In use cases, replace "AddressBook" with "Codii" or "address book"
In use cases, get rid of the entire favourite list thing or change it to black/white/overdue list
In NFR, remove point 7 (log out after some inactivity)

**Comment by khooroko** [comment]
UG:
The "must be positive 1,2,3... "thing is not fixed in nearby command.
the same thing is wrong everywhere else. It's "(e.g. (e.g. 1, 2, 3, …​))" right now.
Perhaps we can write something about the tag colours in the features coming in 2.0 so people don't report it as a bug

**Comment by khooroko** [comment]
Note that in UG, under 2.0 features, it is "two FA". Change it to whatever you change it to in the DG (i.e. either two-FA or 2FA).

**Comment by khooroko** [comment]
UG:
FAQ question 1, the answer should not have "folder" at the end. Also can consider changing "Address Book" there to "address book".

**Comment by khooroko** [comment]
UG:
There is no space before the bracket under nearby

**Comment by khooroko** [comment]
DG:
2FA without the hyphen. The formatting in glossary is also broken for 2FA.
Appendix status bar footer part there is still one instance of "addressbook", change it to "address book".
You have not replaced "AddressBook" with "Codii" or "address book" in use cases -> add person -> extensions yet.

**Review by khooroko — APPROVED** [review]

**Pull request merged**